src/screens/VaccineScreen.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/forbid-prop-types */
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import React from 'react';
+import { BsArrowRightCircleFill } from 'react-icons/bs';
+import '../App.css';
+
+const VaccineScreen = ({ vaccine, getCountry }) => (
+  <>
+    <div className="card inner-wrapper" data-id={vaccine.All.country} data-testid="card">
+      <Link to={`/detail/${vaccine.All.country}`} onClick={getCountry} type="submit">
+        <BsArrowRightCircleFill color="white" />
+      </Link>
+      <h1>{vaccine.All.country}</h1>
+      <h1>{vaccine.All.people_vaccinated.toLocaleString()}</h1>
+    </div>
+  </>
+);
+
+VaccineScreen.propTypes = {
+  vaccine: PropTypes.object.isRequired,
```

Comment:
- [ ] <b>REQUIRED:</b> You can fix the above `eslint` error by using the following line of code on line 21.

```js
VaccineScreen.propTypes = {
  vaccine: PropTypes.shape({ }).isRequired
  }
```

Reply:
Thanks, @rukundoeric. Recommendation applied